### PR TITLE
CP-2846 Release w_transport 3.0.0-rc.2

### PR DESCRIPTION
**Pull request by rmconsole-wf**
Pulls Included in Release:
* [CP-2820 Restore existing MockClient() constructor](https://github.com/Workiva/w_transport/pull/231)
* [CP-2821 Documentation tweaks](https://github.com/Workiva/w_transport/pull/232)
* [Update tests and increase coverage](https://github.com/Workiva/w_transport/pull/233)

Requested by: @evanweible-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/w_transport/compare/3.0.0-rc.1...version-bump-w_transport-3-0-0-rc-2
Diff Between Last Tag and New Tag: https://github.com/Workiva/w_transport/compare/3.0.0-rc.1...3.0.0-rc.2

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
Number of Findings: 0

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: w_transport
-version: 3.0.0-rc.1
+version: 3.0.0-rc.2-rc.1
```
Comment (evanweible-wf) [inline pubspec.yaml]: So close

**Comment by maxwellpeterson-wf** [comment]
+1

**Comment by trentgrover-wf** [comment]
+1

**Comment by maxwellpeterson-wf** [comment]
QA +1

- [x] Dev +1's only needed for Release ticket

Ready for merge and tag.

**Pull request merged**